src/common/math_utils.hpp

```diff
@@ -327,8 +334,9 @@ inline bool eltwise_fwd_preserves_zero(alg_kind_t alg, bool jit_impl = false) {
     using namespace utils;
     const bool preserves_zero = true
         && !one_of(alg, eltwise_linear, eltwise_soft_relu, eltwise_logistic,
-                eltwise_exp, eltwise_clamp, eltwise_not, eltwise_swish)
-        && IMPLICATION(jit_impl, !one_of(alg, eltwise_elu, eltwise_tanh, eltwise_clamp, eltwise_not, eltwise_swish));
+                eltwise_exp, eltwise_clamp, eltwise_not, eltwise_swish, eltwise_hsigmoid)
+        && IMPLICATION(jit_impl, !one_of(alg, eltwise_elu, eltwise_tanh, eltwise_clamp,
+                                             eltwise_not, eltwise_swish, eltwise_hsigmoid));
```

Comment:
`eltwise_fwd_preserves_zero` should return true is f(0) = 0.  Does this condition works for HSigmoid?

Reply:
Fixed, thank you!